code/modules/mob/living/simple_animal/abnormality/aleph/titania.dm

```diff
@@ -107,10 +117,108 @@
 	if(LAZYLEN(potentialmarked))
 		nemesis = pick(potentialmarked)
 
+//Cleaning fairies
+/mob/living/simple_animal/hostile/abnormality/titania/death(gibbed)
+	for(var/mob/living/A in spawned_mobs)
+		A.death()
+	..()
+
+
+//------------------------------------------------------------------------------
+//Fairy Laws
+//------------------------------------------------------------------------------
+
+/mob/living/simple_animal/hostile/abnormality/titania/proc/SetLaw()
+	var/lawmessage
+
+	if(!nemesis || nemesis.stat == DEAD)
+		laws -= "nemesis"
+	nextlaw = pick(laws)
+
+	switch(nextlaw)
+		if("melee")
+			lawmessage = "Thou shalt not hit thy queen with melee attacks."
+		if("ranged")
+			lawmessage = "Thou shalt not hit thy queen with ranged attacks."
+		if("fairy")
+			lawmessage = "Mine fairies are now heartier."
+		if("armor")
+			lawmessage = "Thy queen shalt not be hurt by red damage."
+		if("nemesis")		//This mostly is so that you can tell who the nemesis is, and to encourage them to fight her.
+			lawmessage = "No one but [nemesis] shalt hurt thy queen."
+		if("ranged fairy")
+			lawmessage = "Mine fairies will come to my aid if you strike me with ranged attacks."
+
+	for(var/mob/living/carbon/human/H in range(15, src))
```

Comment:
This will duplicate visible message for the amount of people in range. You don't need it here.